docs/cli-arguments.md

```diff
@@ -60,6 +60,7 @@ Flags:
       --namespaces-denylist string                 Comma-separated list of namespaces not to be enabled. If namespaces and namespaces-denylist are both set, only namespaces that are excluded in namespaces-denylist will be used.
       --node string                                Name of the node that contains the kube-state-metrics pod. Most likely it should be passed via the downward API. This is used for daemonset sharding. Only available for resources (pod metrics) that support spec.nodeName fieldSelector. This is experimental.
       --one_output                                 If true, only write logs to their native severity level (vs also writing to each lower severity level; no effect when -logtostderr=true)
+      --options-config-file string                 Path to the kube-state-metrics options config file
```

Comment:
Can we rename it to `--config` ? makes it easier to discover as a user.

Reply:
(I know there are different config files for custom resource metrics, but this is probably the main config)